### DISCUSSION TITLE
Highlight of the bottom line

### DISCUSSION THREAD

**Discussion by VDuchauffour** [nvim-tree/nvim-tree.lua#1554] [Q&A] [answered]
Hello, I use Lunarvim and I don't know why I'm getting this color shift when nvim-tree is active or not (i.e. when I move from nvim-tree to a buffer). There is a highlight setting for that ? I mean something like `NvimTreeFolderIcon` for example. 
I would like to have the background color inactive all the time.
Figure 1 is when nvim-tree is inactive (i.e. when I'm in a buffer), Figure 2 is when it's active.
Thanks

![Screenshot_2022-07-08_16:47:52_(934x551)](https://user-images.githubusercontent.com/40724343/186725012-70724876-8fd6-4e8a-a0c8-f541a28dcdba.png)
![Screenshot_2022-07-08_16:47:48_(934x551)](https://user-images.githubusercontent.com/40724343/186725026-8dfa7669-c7f7-48e3-9410-4aaafd5578e4.png)

**Comment by alex-courtis** [comment, marked as answer]
It appears to be the status line that is changing colour.

You'll need to configure whatever plugin lunarvim uses for statusline.

**Reply by VDuchauffour** [reply]
Ok thanks, I will look upon my lualine config.

**Reply by alex-courtis** [reply]
See https://github.com/kyazdani42/nvim-tree.lua/issues/424#issuecomment-1100848248
